### PR TITLE
Doctor's Delight looks purple in a beaker, rather than shit brown

### PR DESCRIPTION
**Pull request by 9600bauds**
yep
that's all i have

### CODE REVIEW

**Comment by Shadowmech88** [comment]
If DD is a mixture of orange juice, tomato juice, lime juice, milk, and tricord, wouldn't it make sense for it to look shit brown?

**Comment by Exxion** [comment]
Yes, but none of our other chemical colors make sense anyway
Not even water

**Comment by Kammerjunk** [comment]
Well, I mean, iron's the "beneficial chem" purple colour, even though it's liquid iron. In fact, most beneficial drugs have that light C8A5DC colour. Radium's white but leaves green stains, sulphuric acid is orange, etc

**Comment by Shadowmech88** [comment]
Realistic reagent colors when?

**Comment by Exxion** [comment]
Most of them would look just like water

**Comment by Shadowmech88** [comment]
Is that necessarily bad?

**Comment by Imagebard** [comment]
It would be jarring
Not even our water looks like water

**Comment by Shadowmech88** [comment]
I think our water looks enough like water. It's not light blue anymore, at least.

**Comment by Exxion** [comment]
If your tap water even vaguely resembles the color of water in beakers, it may, in fact, be flammable

**Comment by Intigracy** [comment]
@9600bauds make radium green like when you splash it please.

Iron should look like iron but I don't care about that one as much.

**Comment by 9600bauds** [comment]
How the fuck did this get 10 comments

**Comment by 9600bauds** [comment]
dreamseeker_2016-04-20_17-36-07
dreamseeker_2016-04-20_17-35-11
dreamseeker_2016-04-20_17-37-09

**Comment by Intigracy** [comment]
Thanks fam

**Comment by nuklearcellphoneg** [comment]
> Liquid iron
> Doesn't destroy the glass beaker

¯_(ツ)_/¯

**Comment by Exxion** [comment]
I'm pretty sure glass has a higher melting point than iron
Liquid carbon is what you should really worry about

**Comment by Exxion** [comment]
Actually I'm wrong
Glass and iron have about the same melting point
But carbon is still much higher

**Comment by nuklearcellphoneg** [comment]
All burnt living shit is carbon, no?

**Comment by Exxion** [comment]
Burning carbon is easy
Melting it is not
Its melting point is about 2000 degrees Celsius higher than that of iron
But that's not really relevant to this issue

**Comment by nuklearcellphoneg** [comment]
Ah but it is.

50u inaprovaline and 50u carbon makes bicaridine. :^)

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -1434,7 +1434,7 @@
 	id = "iron"
 	description = "Pure iron is a metal."
 	reagent_state = SOLID
-	color = "#C8A5DC" //rgb: 200, 165, 220
+	color = "#666666" //rgb: 102, 102, 102
```
Comment (Intigracy) [inline code/modules/reagents/Chemistry-Reagents.dm:1437]: fukkin satan

**Comment by Shadowmech88** [comment]
I always figured the "solid" reagents like iron and carbon were just ground powders in the beaker, rather than molten liquids.
Hence why you can obtain plasma from grinding plasma bars.

File: code/modules/reagents/Chemistry-Reagents.dm (hunk 1 above)
Comment (9600bauds) [inline code/modules/reagents/Chemistry-Reagents.dm:1437]: I grabbed a random grey and hit "closest web-compatible" in Photoshop

**Comment by nuklearcellphoneg** [comment]
But then you splash it all over the floor.

**Comment by Exxion** [comment]
Well, salt and pepper are also both treated as liquids
Rice too
And dry ramen
We only even have liquid reagents

**Comment by 9600bauds** [comment]
_GLUGLUG_ You swallow some of the contents of flour sack.

**Comment by nuklearcellphoneg** [comment]
GLUG GLUG YOU SWALLOW THE SALT/PEPPER

**Comment by FudgePucker** [comment]
Are you guys really trying to think of our chem system as realistic?

I can pour 50 units of liquid hydrogen into an open beaker and the room doesn't explode.

**Comment by nuklearcellphoneg** [comment]
Just making conversation

**Pull request merged**